.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,70 @@
-**Please provide the following information**
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+
+Be honest!
+-->
+
+- [ ] I've read the **issue reporting guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good report with the required information
```

Comment:
Skipping contractions is probably better for the minimally English speaking crowd. Regardless, your contraction usage is inconsistent, line 26 doesn't use contractions while the rest of your cases do.

Reply:
Thanks, fixed.